docker/deployed/testnet/api/config.json

```diff
@@ -2,7 +2,7 @@
     "Impl": "mesa",
     "HTTP": {
         "Port": "8080",
-        "RateLimInterval": "1s",
+        "RateLimInterval": "4s",
```

Comment:
Looks like there was a person from HackFS getting 429. Do you think we should still keep this rate until we are more confident that we can increase, or should we try to be a bit looser? My gut says we can try 2s (hopefully reducing 429 for devs and not shooting ourselves in the foot).